packages/slice-machine/src/icons/ReusableIcon.tsx

```diff
@@ -1,23 +1,17 @@
-import { FC } from "react";
+import { FC, SVGProps } from "react";
 
-export const ReusableIcon: FC = () => (
-  <svg
-    width="32"
-    height="32"
-    viewBox="0 0 32 32"
-    fill="none"
-    xmlns="http://www.w3.org/2000/svg"
-  >
-    <path
-      d="M8 28H24C25.1046 28 26 27.1046 26 26V9L21 4H8C6.89543 4 6 4.89543 6 6V26C6 27.1046 6.89543 28 8 28Z"
-      fill="#6E56CF"
-    />
-    <path d="M23 9H26L21 4V7C21 8.10457 21.8954 9 23 9Z" fill="#9E8CFC" />
-    <path
-      fillRule="evenodd"
-      clipRule="evenodd"
-      d="M16.0912 12C13.2794 12 11 14.2794 11 17.0912C11 19.903 13.2794 22.1824 16.0912 22.1824C17.9763 22.1824 19.6215 21.1577 20.501 19.6372C20.6563 19.3688 20.7878 19.0849 20.8926 18.7883C20.9846 18.5279 20.8482 18.2422 20.5878 18.1502C20.3275 18.0582 20.0418 18.1947 19.9498 18.455C19.8657 18.693 19.7601 18.9209 19.6354 19.1365C18.9273 20.3606 17.6049 21.1824 16.0912 21.1824C13.8317 21.1824 12 19.3507 12 17.0912C12 14.8317 13.8317 13 16.0912 13C17.4478 13 18.6507 13.66 19.3955 14.6782H17.6216C17.3455 14.6782 17.1216 14.9021 17.1216 15.1782C17.1216 15.4544 17.3455 15.6782 17.6216 15.6782H20.2998C20.576 15.6782 20.7998 15.4544 20.7998 15.1782V12.5C20.7998 12.2239 20.576 12 20.2998 12C20.0237 12 19.7998 12.2239 19.7998 12.5V13.6032C18.8717 12.6167 17.5536 12 16.0912 12Z"
-      fill="white"
-    />
+export const ReusableIcon: FC<SVGProps<SVGSVGElement>> = (props) => (
+  <svg viewBox="0 0 20 24" xmlns="http://www.w3.org/2000/svg" {...props}>
+    <g fill="none">
```

Comment:
Good catch. I re-exported the SVGs without the extra embedded whitespace. The whitespace was causing sizing issues with `Table` since the component requires a set column template.

I ran the re-exported SVG through SVGO, and I guess it figured the `<g>` was necessary. I removed them manually and the icons seem to work the same as with them.